### DISCUSSION TITLE
PROPOSAL: Utilize dev version numbers

### DISCUSSION THREAD

**Discussion by ddsjoberg** [pharmaverse/admiral#2030] [Ideas]
It's common practice use dev version numbers between releases, e.g. `0.11.1.9000`. The usethis package has functions to assist in incrementing version numbers that update both the `DESCRIPTION` file and `NEWS.md`, i.e. `usethis::use_version()`. When we use this function, we never need to manually updated the NEWS headers nor the version number in the DESCRIPTION file. It is also a clear signal to anyone visiting the repo whether the version they are looking at is released or dev.

Not urgent, but let's discuss sometime in the next couple of weeks.

**Comment by bms63** [comment]
We are implementing and tracking this in another PR https://github.com/pharmaverse/admiral/pull/2100.  Going to close out.
